recipes/pykalman/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.9.5" %}
+
+package:
+  name: pykalman
+  version: {{ version }}
+
+source:
+  fn: pykalman-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pykalman/pykalman-{{ version }}.tar.gz
+  md5: 8baa2921cdfd2a5193ac67e23c2f7e5e
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - numpy
+    - scipy
+
+test:
+  imports:
+    - pykalman
+    - pykalman.datasets
+    - pykalman.sqrt
+    - pykalman.sqrt.tests
+    - pykalman.tests
```

Comment:
That happens in a lot of our feedstocks due to the `conda-skeleton` boilerplate. Let's get this into a feedstock and the discussion about further testing can happen there.